### PR TITLE
Improving documentation starting from circleci.

### PR DESCRIPTION
**Pull request by NathanDotTo**
As discussed elsewhere, I am working out from the .circleci towards updated documentation for build. 

One of the key questions to start with is how to set the required variables. There is a `bin/minimal_env.sh` script that could be used uniformly. The CircleCI documentation suggests that the variables need to be set in the CircleCI env though.

Which is the best approach? Ideally, we would have a single place to set variables, but that might not be desirable for reasons to do with how CircleCI works that I am not aware of.

Also, we have, in minimal_env:

```
export GOOGLE_CREDENTIALS=$(cat key.json | tr -s '\n' ' ')
```

Where the `key.json` is explicitly .gitignore'd. Ideally, we would have an example `key.json`, or at least some explanation of where to get that file from.

### CODE REVIEW

File: .circleci/README.md [hunk 1]
```diff
@@ -2,7 +2,9 @@
 
 ## Variables
 
-See the list of possible environment variables in the follwoing table. They can be set in the project settings, or part of an organization-wide context. Without setting the **required** variables, the pipeline will fail.
+See the list of possible environment variables in the following table. They can be set in the project settings, or part of an organization-wide context. Without setting the **required** variables, the pipeline will fail.
+
+Note that the required variables are also set in the [minimal_env.sh](../bin/minimal_env.sh) script.
```
Comment (ljakab) [inline .circleci/README.md:7]: I would not add this line to the CircleCI documentation specifically, as CircleCI doesn't source this file. `./bin/minimal_env.sh` is only relevant for use of these tools interactively on the command line. Setting variables in CircleCI is done via the UI (and maybe the API, but haven't looked into that) in the **Project Settings** --> **Environment Variables** menu.

**Review by ljakab — CHANGES_REQUESTED** [review]
The environment variables are mostly secrets. Since we don't have proper secret management yet, this is the workaround. We use environment variables, so that we don't have to commit secrets into the Git repository.

**Comment by NathanDotTo** [comment]
Please see updates. Have I explained the OPA policy aspects properly in the new README?

**Review by ljakab — APPROVED** [review]
LGTM, thank you!

**Pull request merged**